prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -300,33 +301,33 @@ private void UpdateRemoteAccessLocations(Enrollee dbEnrollee, EnrolleeUpdateMode
                 _context.Remove(location);
             }
 
-            if (updateEnrollee?.RemoteAccessLocations != null && updateEnrollee?.RemoteAccessLocations.Count() != 0)
-            {
-                var remoteAccessLocations = new List<RemoteAccessLocation>();
+            if (updateEnrollee?.RemoteAccessLocations == null || !(updateEnrollee.RemoteAccessLocations).Any()) return;
```

Comment:
Dont need the `?` on the left or the `()` on the right. I prefer braces on the return.